### PR TITLE
Add post for Continuous Queries, Part I.

### PR DESCRIPTION
**Pull request by toddboom**

### CODE REVIEW

File: content/blog/2015/10/21/continuous_queries_part_1.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+So, let’s get started.
+
+## Introducing continuous queries in InfluxDB
+
+If you come from an RDBMS background, continuous queries are conceptually very similar to materialized views, where expensive query results are pre-computed and stored, instead of being computed on-the-fly. Continuous queries in InfluxDB are powerful and useful for several different use-cases, mainly around improving the experience around data analysis as described below.
+
+### Viewing data efficiently with continuous queries
+
+In a dashboard application, metric data is typically presented in the form of graphs. Typically, these metrics are downsampled before they are displayed. This means that, they are either  aggregated around a particular time interval or simply the Nth data point is picked in the metric series.
```
Comment (otoolep) [inline content/blog/2015/10/21/continuous_queries_part_1.md]: Looks and extra space between "either" and "aggregated".

I think it should be "picked from"?

**Comment by otoolep** [comment]
Looks could, didn't see any real typos.

File: content/blog/2015/10/21/continuous_queries_part_1.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Introducing continuous queries in InfluxDB
+
+If you come from an RDBMS background, continuous queries are conceptually very similar to materialized views, where expensive query results are pre-computed and stored, instead of being computed on-the-fly. Continuous queries in InfluxDB are powerful and useful for several different use-cases, mainly around improving the experience around data analysis as described below.
+
+### Viewing data efficiently with continuous queries
+
+In a dashboard application, metric data is typically presented in the form of graphs. Typically, these metrics are downsampled before they are displayed. This means that, they are either  aggregated around a particular time interval or simply the Nth data point is picked in the metric series. 
+
+With continuous queries in InfluxDB, you can downsample application data and improve the end user experience for your users.  For example, as shown in the figure below, the input graph is downsampled, and the output captures every 10th point in the input graph.
```
Comment (beckettsean) [inline content/blog/2015/10/21/continuous_queries_part_1.md]: It seems odd to provide a graph that shows a function InfluxDB cannot do (InfluxDB cannot downsample by point count, only by time window.)

**Comment by beckettsean** [comment]
@toddboom @jginfluxdata what happened with this? This open PR was never merged and my comment wasn't addressed, but the blog post is live. I can't find the PR where it was added, it looks like a direct commit from @toddboom.

I'm not super worried about it, more concerned that I don't understand how to give feedback for future blog posts. With external writers composing them I would like to have some editorial approval over them. I don't quite trust that the examples were tested, for instance. Maybe they were, maybe they weren't, but when I last talked with the author I could tell he had not been testing his queries.

**Pull request closed without merging**